README.md

```diff
@@ -1,4 +1,44 @@
-# Open Autonomy
+<h1 align="center">
+    <b>Open Autonomy Framework</b>
+</h1>
+
+<p align="center">
+  <a href="https://pypi.org/project/open-autonomy/">
+    <img alt="PyPI" src="https://img.shields.io/pypi/v/open-autonomy">
+  </a>
+  <a href="https://pypi.org/project/open-autonomy/">
+    <img alt="PyPI - Python Version" src="https://img.shields.io/pypi/pyversions/open-autonomy">
+  </a>
+  <a>
+    <img alt="PyPI - Wheel" src="https://img.shields.io/pypi/wheel/open-autonomy">
+  </a>
+  <a href="https://github.com/valory-xyz/open-autonomy/blob/main/LICENSE">
+    <img alt="License" src="https://img.shields.io/pypi/l/open-autonomy">
+  </a>
+  <a href="https://pypi.org/project/open-autonomy/">
+    <img alt="Downloads" src="https://img.shields.io/pypi/dm/open-autonomy">
+  </a>
+</p>
+<p align="center">
+  <a href="https://github.com/valory-xyz/open-autonomy/actions/workflows/main_workflow.yml">
+    <img alt="Sanity checks and tests" src="https://github.com/valory-xyz/open-autonomy/workflows/main_workflow/badge.svg?branch=main">
+  </a>
+  <a href="">
```

Comment:
The codecov link is empty as in open-aea's header. I guess it's ok since the image shows correctly and I didn't find a link that this should point to.